packages/flutter/lib/src/cupertino/checkbox.dart

```diff
@@ -166,7 +199,7 @@ class CupertinoCheckbox extends StatefulWidget {
   final OutlinedBorder? shape;
 
   /// The width of a checkbox widget.
-  static const double width = 18.0;
+  static const double width = 14.0;
```

Comment:
Small not on the background of this, nothing that needs to change: the size of 18 was used to match the size of the Material checkbox. This was from when we where concerned about adaptive widgets having different dimensions, causing the UI to "shift" around unexpectedly. We are less concerned about this now, and it's better to have the Cupertino widget have the native default value. If the adaptive widget becomes an issue, we can wrap the Cupertino widget with a `Transform.scale` in the adaptive delegation.